Monika After Story/game/script-food.rpy

```diff
@@ -0,0 +1,158 @@
+init 5 python:
+    addEvent(Event(persistent.event_database,eventlabel="monika_imeating",
+                    category=['you'],prompt="I'm Eating/Drinking",pool=True,unlocked=True))
```

Comment:
I really don't think you should use a slash in the prompt. I did recommend to have 2 separate prompts and separate the food and drinks. That way, you can have more specific dialogue leading up than just having something general. 

Currently, Monika asks if you're eating, and if you're drinking, you're not really eating.